### PR TITLE
update a project domain: staticaly.com to statically.io

### PR DESCRIPTION
**Pull request by fransallen**
also added thedev.id and Statically Screenshot

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1389,6 +1390,7 @@ Table of Contents
   * [smsreceivefree.com](https://smsreceivefree.com/) — Provides free temporary and disposable phone numbers.
   * [simplebackups.io](https://simplebackups.io/) — Backup automation service for servers and databases (MySQL, PostgreSQL, MongoDB) stored directly into cloud storage providers (AWS, DigitalOcean, Backblaze...). Provides free plan for 1 backup.
   * [SnapShooter](https://snapshooter.com/) — Backup solution for DigitalOcean, AWS, LightSail, Hetzner and Exoscale, with support for direct database, file system and application backups to s3 based storage. Provides a free plan with daily backups for one resource.
+  * [thedev.id](https://thedev.id) — A free domain for developers.
```
Comment (FatGrizzly) [inline README.md:1393]: Isn't it a subdomain?(not ripienaar, but thought to put it up here)

Comment (fransallen) [reply README.md:1393]: Yeah it's a thedev.id subdomain. I'll update it to make it clearer.

File: README.md [hunk 2]
```diff
@@ -1324,6 +1324,7 @@ Table of Contents
   * [ScreenshotAPI.net](https://screenshotapi.net/) - Screenshot API use one simple API call to generate screenshots of any website. Build to scale and hosted on Google Cloud. Offers 100 free screenshots per month.
   * [screenshotlayer.com](https://screenshotlayer.com/) — Capture highly customizable snapshots of any website. Free 100 snapshots/month
   * [screenshotmachine.com](https://www.screenshotmachine.com/) — Capture 100 snapshots/month, png, gif and jpg, including full-length captures, not only home page
+  * [Statically Screenshot](https://statically.io/docs/using-screenshot/) — A free screenshot API to convert web to image or PDF. Supports desktop and mobile view.
```
Comment (ripienaar) [inline README.md:1327]: think we can just update the one entry with additional features

Comment (fransallen) [reply README.md]: no problem, I revert the addition for now

**Comment by ripienaar** [comment]
thank you

**Pull request merged**